changelog/20.0/20.0.0/summary.md

```diff
@@ -7,7 +7,6 @@
   - **[Deletions](#deletions)** 
     - [`--vreplication_tablet_type` flag](#vreplication-tablet-type-deletion)
     - [Pool Capacity Flags](#pool-flags-deletion)
-    - [MySQL binaries in the vitess/lite Docker images](#vitess-lite)
```

Comment:
In the `v19.0.0` release notes we say the following:
> The `mysqld` binary is now deprecated in the `vitess/lite` Docker image and will be removed in a future release.

Thus, other than removing the release notes we added in `v20` I don't think there is anything else needed here.

We are removing the `mysql-57` and `percona-57` images but that's a side effect of dropping support for 5.7 in `v19`